Insert into database functionality

angapinski opened the pull request:
see commit descriptions for details

Commit:
SQL Insert statement, front end

added variable distance to code and sql table

Commit:
Merge branch 'master' into insertIntoDatabaseFunctionality

Commit:
Finished insert Functionality

changed type dropdown to integer values
corrected misspelled class names
added hidden note input type to send formatted notes string
corrected names in SQL database and added distance column as INT NOT NULL

aperdew commented on WebContent/WOH-addRestaurant.jsp:
```diff
@@ -53,20 +53,20 @@
 						<label class="control-label col-sm-2" for="WOH-addRestaurant-distanceInput">Distance</label>
 							<div class="col-sm-10">
-	         					<input type = "text" class="form-control WOH-addRestaurant-distanceInput" name = "distance">
+	         					<input type = "text" class="form-control WOH-addRestaurant-distanceInput" value = 0 name = "distance">
```
change this to input type range.
<input type="range" name="distance" min="2" max="200">
http://www.eyecon.ro/bootstrap-slider/
this website has some cool documentation for how to dress it up.

aperdew commented on WebContent/WOH-addRestaurant.jsp:
```diff
@@ -31,13 +31,13 @@
+	         					<input type = "text" class="form-control WOH-addRestaurants-restaurantNameInput" name = "name">
 	         				</div>
 	         			</div>
 	         			
```
Change the price input into a drop down with options: $,$$,$$$,$$$$.  within formatData(), use a switch case to change these String values into their corresponding int values.

aperdew commented on WebContent/WOH-addRestaurant.jsp:
```diff
@@ -87,8 +87,9 @@
-  							<textarea class="form-control WOH-addRestaurants-notesInput" rows="5" name="notes"></textarea>
+							<input type="hidden" name="notes" class="WOH-addRestaurant-notesInput">
+  							<label for="WOH-addRestaurants-notesTextArea">Notes</label>
+  							<textarea class="form-control WOH-addRestaurants-notesTextArea" rows="5" name="notesInput"></textarea>
```
For right now, this is okay, but later, I want these notes to go into a table so it looks more like comments on Facebook or something like that.  this will allow the user to easily edit and delete notes.

aperdew commented on WebContent/WOH-addRestaurant.jsp:
```diff
@@ -53,20 +53,20 @@
-		    						<option value = "fastFood" >Fast Food</option>
-		    						<option value = "sitDown" >Sit-Down</option>
-		    						<option value = "dessert" >Dessert</option>
+		    						<option selected="selected" value = "1" >Fast Food</option>
```
This is good for now, but in the future, I want you to set a placeholder for all of the drop downs.
<option value="" disabled selected>Select your option</option>
you could do something like this.

aperdew reviewed (commented):
Everything looks great, just a few minor ui changes and a few notes for the future.

Commit:
addRestaurant Changes

Many of the changes we discussed, checkbox panel on right hand side, added search bar

Commit:
Working Slider, Cosmetics

slider is functional and converts miles to meters for Google API integration.
Cosmetic Changes to finalize addRestaurant page.

Commit:
Merge branch 'master' into insertIntoDatabaseFunctionality
